kroxylicious-kms-provider-aws-kms/src/test/java/io/kroxylicious/kms/provider/aws/kms/AwsKmsTest.java

```diff
@@ -150,7 +150,9 @@ void withMockAwsWithSingleResponse(String response, int statusCode, Consumer<Aws
             var address = httpServer.getAddress();
             var awsAddress = "http://127.0.0.1:" + address.getPort();
             var config = new Config(URI.create(awsAddress), new InlinePassword("access"), new InlinePassword("secret"), "us-west-2", null);
-            var service = new AwsKmsService().buildKms(config);
+            var awsKmsService = new AwsKmsService();
+            var initData = awsKmsService.initialize(config);
+            var service = awsKmsService.buildKms(initData);
```

Comment:
Whats your thinking around why initialize is a separate and external step to `buildKms`? Isn't it just the first thing the `kms` impl would do?